examples/interaction/js_callbacks/setvalue.py

```diff
@@ -1,3 +1,14 @@
+''' A visualization of buttons in bokeh.models. This example demonstrates
+changing the value of an object when a certain event (like clicking of a button)
+happens.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.io.show, bokeh.models.Button, Bokeh.models.SetValue
+    :refs: :ref:`ug_interaction_js_callbacks_setvalue`
+    :keywords: javascript SetValue
+'''
+
+
 from bokeh.io import show
```

Comment:
no blank lines between docstring and code start
```suggestion
from bokeh.io import show
```